autometa/taxonomy/ncbi.py

```diff
@@ -188,96 +184,82 @@ def get_lineage_dataframe(self, taxids, fillna=True):
             how='left',
             left_on=<taxid_column>,
             right_index=True)
```

Comment:
The `NOTE` here should probably be formatted for sphinx readthedocs and the merge example you can write as a python snippet.

See here for an example:

https://github.com/KwanLab/Autometa/blob/ddc2bf951d6c4a8c29e9e2301e15a762327b0267/autometa/common/external/prodigal.py#L182-L206

Which is formatted in [autometa.rtfd.io/prodigal](https://autometa.readthedocs.io/en/latest/API_documentation/autometa.common.external.html#autometa.common.external.prodigal.contigs_from_headers)